test/project_types/script/layers/infrastructure/fake_extension_point_repository.rb

```diff
@@ -22,12 +26,20 @@ def get_extension_point(type)
           end
         end
 
+        def extension_points
+          @cache.values
+        end
+
         def extension_point_types
-          %w(discount unit_limit_per_order)
+          @cache.keys
```

Comment:
This didn't necessarily need to change but it didn't seem right to have this method use a hardcoded list and ignore `create_extension_point` calls.